s/elements/bindings-editor/element.ts

```diff
@@ -36,7 +36,12 @@ export const NubBindingsEditor = element({
 		</div>
 
 		${showTextEditor
-			? TextEditorPanelView({bindings})
+			? TextEditorPanelView({
+				bindings,
+				onClickSave(draft) {
+					context.updateBindings(draft)
+				},
+			})
```

Comment:
here i've implemented the text editor saving changes

Reply:
where is that updateBindings ?

Reply:
I see its in the context element, but how does it update bindings ?